demo/tmail/mailetcontainer.xml

```diff
@@ -96,13 +96,6 @@
             <mailet match="All" class="com.linagora.tmail.mailets.TmailLocalDelivery">
                 <consume>false</consume>
             </mailet>
-
-            <!-- Put the OpenAIMailet after LocalDelivery so the GPT reply would come after the asking question -->
-            <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.AIBotMailet">
-                <apiKey>demo</apiKey>
-                <botAddress>gpt@tmail.com</botAddress>
-                <model>gpt-4o-mini</model>
-            </mailet>
```

Comment:
this mailet caused the server to fail at booting, removing that fixed the problem for me. but i don't know if we should remove it entirely though.